projects/vdk-plugins/vdk-notebook/src/vdk/plugin/notebook/notebook.py

```diff
@@ -82,6 +82,7 @@ def register_notebook_steps(file_path: Path, context: JobContext):
                         )
                         notebook_steps.append(step)
                         context.step_builder.add_step(step)
+                        context.step_builder.sort_steps()
```

Comment:
how many times is the sorting being executed, seems in a for cycle? it should be executed once at the end of init job phase

Reply:
fixed